frontend/css/layout.css

```diff
@@ -114,3 +279,10 @@ li a {
 .subtitle {
     text-align: center;
 }
+
+@media (max-width: 480px) {
+    .mobile-grid {
```

Comment:
Rather than applying a separate class, could we apply `grid-template-columns: 1fr;` within the existing class specifying the grid, given that the viewport is smaller than `480px`?

Reply:
Thanks Sara - i've updated the media and removed the additional mobile-grid class